src/blockchain/assets.md

```diff
@@ -0,0 +1,83 @@
+# Assets
```

Comment:
Do we still have `Asset Definition` in Iroha MVP? If so a kind of separation between those terms should be explained explicitly